mod-audit-server/src/test/java/org/folio/TestSuite.java

```diff
@@ -250,10 +250,6 @@ class AuditDataImplApiTestNested extends AuditDataImplApiTest {
   class CirculationLogsImplApiTestNested extends CirculationLogsImplApiTest {
   }
 
-  @Nested
-  class InventoryEventServiceImplNestedTest extends InventoryEventServiceImplTest {
-  }
-
```

Comment:
Why is this removed?

Reply:
It now included in unit test run instead of integration test suit.